climakitae/core/data_load.py

```diff
@@ -281,19 +281,28 @@ def set_subarea(boundary_dataset):
             )
         ds_region = [geom]
     elif area_subset != "none":
-        shape_index = int(selections._geography_choose[area_subset][cached_area])
+        shape_indicies = list(
+            {
+                key: selections._geography_choose[area_subset][key]
+                for key in cached_area
+            }.values()
+        )
         if area_subset == "states":
-            shape = set_subarea(selections._geographies._us_states)
+            shape = set_subarea(selections._geographies._us_states, shape_indicies)
```

Comment:
I added `shape_indicies` as arguments into the `set_subarea` function (not formerly added), but I thought for clean variable referencing, it was good to pass the argument through with the function, instead of referencing the variable outside of the function itself. Let me know what you think, and/or if I should leave it as is instead.

Reply:
Indices has a typo in it throughout (there's an extra "i") - minor though but would need to addressed throughout!

Reply:
🤦‍♂️ ... I am a fool... will fix.